api/src/test/scala/uk/ac/wellcome/platform/api/services/WorksServiceTest.scala

```diff
@@ -4,18 +4,21 @@ import com.sksamuel.elastic4s.ElasticDsl._
 import org.junit.Test
 import org.scalatest.{AsyncFunSpec, Matchers}
 import uk.ac.wellcome.models.{IdentifiedWork, SourceIdentifier, Work}
-import uk.ac.wellcome.platform.api.models.Record
+import uk.ac.wellcome.platform.api.models.DisplayWork
 import uk.ac.wellcome.test.utils.IndexedElasticSearchLocal
 import uk.ac.wellcome.utils.JsonUtil
 
-class ElasticSearchServiceTest
+class WorksServiceTest
```

Comment:
This replaces the `ElasticSearchServiceTest` - and exercises the code therein. I'm unsure as to whether we require another set of tests for the `ElasticSearchService` (we probably do).